src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -349,9 +349,10 @@ public void buildsDomNode() throws Exception {
      */
     @Test
     public void comparesToAnotherDocument() throws Exception {
+        final String xml = "<hi><dude>  </dude></hi>";
         MatcherAssert.assertThat(
-            new XMLDocument("<hi>\n<dude>  </dude></hi>"),
-            Matchers.equalTo(new XMLDocument("<hi><dude>  </dude></hi>"))
+            new XMLDocument(xml),
```

Comment:
Not really. Now that the String `xml` doesn't get automatic indentation, the document with the `\n` is NOT equal to the document without the `\n`. In order for equals to be true, we have to compare instances of XMLDocument constructed with the same String.